docs/topics/signals.txt

```diff
@@ -189,6 +189,20 @@ In some circumstances, the code connecting receivers to signals may run
 multiple times. This can cause your receiver function to be registered more
 than once, and thus called multiple times for a single signal event.
 
+As Django has evolved, this issue has generally less risk of presenting itself.
+Before ``AppConfig.ready`` was introduced to Django, it was possible that
+during the Django setup process during normal operation or from a management
+command, the handlers.py file would be imported multiple times, and thus,
```

Comment:
- Suggestion to add a full stop after 'multiple time'. (I think this long sentence needs breaking up to help readability, here is where I think a break could take place with the current wording)

- Suggestion to change 'thus' to 'so' or 'therefore'

Reply:
Tried out something like that on my change. Please let me know!